var/spack/repos/builtin/packages/py-enum34/package.py

```diff
@@ -29,15 +29,13 @@ class PyEnum34(PythonPackage):
     """Python 3.4 Enum backported to 3.3, 3.2, 3.1, 2.7, 2.6, 2.5, and 2.4."""
 
     homepage = "https://pypi.python.org/pypi/enum34"
-    url      = "https://pypi.python.org/packages/source/e/enum34/enum34-1.1.6.tar.gz"
```

Comment:
Can you convert this package to use `pypi.io`? I confirmed that it works. Also remove the URL from the version directive below.